community/releases/release-manager-runbook/index.md

```diff
@@ -10,11 +10,21 @@ sidebar:
 
 ## A Quick Review of Release Process
 Here's a quick review of what the release process looks like from the community perspective:
-- This is the [expected release cadence](/community/releases/release-cadence/). 
+- This is the [expected release cadence](/community/releases/release-cadence/).
 - The [release calendar](/community/releases/release-cadence/#upcoming-releases) is awesome. It gives you an agenda with the expected duties.
 - Here's what the project expects all contributors to do for [backports/patches](/community/contributing/releasing/#release-branch-patch-criteria).
 
 
+## Verify You Have Access
+- You've been included in [release-manager@spinnaker.io](mailto:release-manager@spinnaker.io) group by contacting a
+member of the TOC or SC.
+- Into the [spinnaker-announce@googlegroups.com](mailto:spinnaker-announce@googlegroups.com) group.
```

Comment:
Not sure what these items are trying to do. Are you saying "verify you've been added into the spinnaker-announce@..."?
And "Verify you've have access to the Jenkins UI"?

If so, I would precede the list with:
"Verify you have access to the following:

- You're a member of the release-manager@spinnaker.io group and the spinnaker-announce@googlegroups.com group. Contact a member of the TOC..."
- You have access to the Jenkins UI
- You can SSH into Jenkins

...and so on.